dashboard/agent.py

```diff
@@ -90,9 +90,14 @@ async def run(self):
         async def _check_parent():
             """Check if raylet is dead."""
             curr_proc = psutil.Process()
+            ppid = os.environ.get("RAY_NODE_PID")
+            if ppid is not None:
```

Comment:
Should we assert that this is not None?

Reply:
Good suggestion. The ppid will be a required arg.